script/nbb_tests.clj

```diff
@@ -98,13 +98,19 @@
                    @result))])))))
 
 (deftest classpath-test
-  (let [deps '{com.github.seancorfield/honeysql {:git/tag "v2.0.0-rc5" :git/sha "01c3a55"}}
-        _ (deps/add-deps {:deps deps})
-        cp (cp/get-classpath)]
-    (is (= ["SELECT foo FROM bar WHERE baz = ?" 2]
-           (nbb "--classpath" cp
-                "-e"
-                "(require '[honey.sql :as sql]) (sql/format {:select :foo :from :bar :where [:= :baz 2]})")))))
+  (testing "passing classpath cli arg"
+    (let [deps '{com.github.seancorfield/honeysql {:git/tag "v2.0.0-rc5" :git/sha "01c3a55"}}
+          _ (deps/add-deps {:deps deps})
+          cp (cp/get-classpath)]
+      (is (= ["SELECT foo FROM bar WHERE baz = ?" 2]
+             (nbb "--classpath" cp
+                  "-e"
+                  "(require '[honey.sql :as sql]) (sql/format {:select :foo :from :bar :where [:= :baz 2]})")))))
+  (testing "add `:paths` from nbb.edn to classpath"
+    (nbb {:dir "test-scripts/paths-test"} "runner.cljs"))
```

Comment:
There is a missing test assertion here

Reply:
@borkdude I'm not sure what I'd assert. The test fails (throws) before adding the new code in `main.cljs`.

Reply:
Just wrap `(is ...)` around it.